generate/writeToJsonFile.js

```diff
@@ -0,0 +1,82 @@
+import path from 'path'
+import fs from 'fs'
+
+const JsonStringifyParameters = {
+  replacer: null,
+  space: 4,
+}
+
+
+export function writeJsonArray(arrayNewItems, config, applyFilter = false) {
```

Comment:
Yes that was the origin aim.
I will rewrite all json files as you suggest so my code will simplify.